fdupes.c

```diff
@@ -1083,6 +1109,27 @@ int main(int argc, char **argv) {
     case 'R':
```

Comment:
Am I missing something or there is a missing switch statement? For everithing else seems a good pull request, too bad that this wasn't merged